core/lsan.suppress

```diff
@@ -6,6 +6,7 @@
 leak:pcap_alloc_pcap_t
 leak:pcap_check_header
 leak:pcap_open_offline_common
+leak:pcap_create_common
```

Comment:
I think the leak is in our code: if I call `pcap_close(handle)` in `TcpFlowReconstructTest::SetUp()`, the LeakSanitizer is happy.

If you confirm that, maybe we can remove all the pcap_* suppressions

Reply:
Ah, neat. Will amend after reproducing

Reply:
I think what @ddiproietto meant was calling `pcap_close(handle)` at the end of `SetUp()`, not `TearDown()`. Since `handle` is only used in the setup method it doesn't need to be a class member...

Reply:
from offline discussion with @melvinw: it is for exception safety.

btw, why are we removing the three existing pcap functions?

Reply:
While debugging this, I noticed that they weren't being tripped anymore (may or may not be a result of the added `pcap_close()`) Happy to put them back if needed

Reply:
Ah, I see! @ddiproietto said that you may be able to remove all `pcap_*` functions. Do you still see `pcap_create_common` in the leak report?

Reply:
Yup, we (mostly @ddiproietto) are debugging the last source of the `pcap_create_common()` leak. There's a test case in `pcap_handle_test` that causes the leak under __very__ specific reproduction conditions.

Reply:
The `leak:pcap_open_offline_common` was necessary because of the bug in our code, Now that we fix the bug, I think we can remove it.

I guess the rest of them are necessary on older versions of libpcap (i.e. the one packaged in ubuntu16.04) that don't include commit the-tcpdump-group/libpcap@cd46a74cf492 ("Use pcap_close() if pcap_create_handle() fails.")

Sorry, it was working on my system (just because I had another version of tcpdump), and I hoped we could remove the suppressions

How about keeping the suppressions (except open_offline_common) and adding a link to the commit that fixes the problem in libpcap, so that I can remember it next time I look at the suppressions? 😄

Sorry, I'm begin way too picky on this!

Reply:
Sounds good to me. Will put them back

Reply:
Alrighty, added some old ones back and updated the comment